packages/custom-resource-bindings/scripts/build-and-minify-sources.js

```diff
@@ -0,0 +1,42 @@
+const esbuild = require('esbuild');
+const fs = require('fs');
+const path = require('path');
+
+const entryPoints = [];
+function recFolderStructure(fileOrDir) {
+  if (fs.statSync(fileOrDir).isDirectory()) {
+    const items = fs.readdirSync(fileOrDir);
+    for (const i of items) {
+      console.log(i);
+      recFolderStructure(path.join(fileOrDir, i));
+    }
+  } else {
+    if (path.extname(fileOrDir) === '.ts') {
+      entryPoints.push(fileOrDir);
+    }
+  }
+}
+
+const bindingsDir = path.join(__dirname, '..', 'lib');
+const testDir = path.join(__dirname, '..', 'test');
+
+recFolderStructure(bindingsDir);
+recFolderStructure(testDir);
```

Comment:
Why are we building the tests?

Reply:
I think maybe do a regular `tsc` pass, and a regular test pass, and independently `esbuild` the JavaScript into minified bundles?